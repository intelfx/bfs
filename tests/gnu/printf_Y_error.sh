cd "$TEST"
mkdir foo
ln -s foo/bar bar

chmod -x foo
defer chmod +x foo

! bfs_diff . -printf '(%p) (%l) %y %Y\n'
