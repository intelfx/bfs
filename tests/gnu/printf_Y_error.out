(.) () d d
(./bar) (foo/bar) l ?
(./foo) () d d
